service/observiq-otel-collector

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+# observIQ OTEL daemon
+# chkconfig: 2345 99 05
+# description: observIQ's distribution of the OpenTelemetry collector
+# processname: observiq-otel-collector
+# pidfile: /var/run/observiq-otel-collector.pid
+
+# Source function library.
+if [ -e /etc/init.d/functions ]; then
+  STATUS=true
+  . /etc/init.d/functions
+fi
+
+if [ -e /lib/lsb/init-functions ]; then
+  PROC=true
+  . /lib/lsb/init-functions
+fi
+
+# Pull in sysconfig settings
+[ -f /etc/sysconfig/observiq-otel-collector ] && . /etc/sysconfig/observiq-otel-collector
+
+PROGRAM=observiq-otel-collector
+LOCKFILE=/var/lock/$PROGRAM
+OIQ_OTEL_COLLECTOR_HOME=/opt/observiq-otel-collector
+OIQ_OTEL_COLLECTOR_STORAGE=/opt/observiq-otel-collector/storage
+HTTP_PROXY=
+HTTPS_PROXY=
+# set this to a user such as observiq-otel-collector, to run as that user
+# observiq-otel-collector user is created during installation
+RUNUSER=
```

Comment:
Should we set this to `observiq-otel-collector` instead of leaving it blank?

Reply:
It will default to `root` if `RUNUSER` is unset, which is what we want. It feels clunky, but if you look at the `if` condition, it will use `su` to run the process as `$RUNUSER`, if set.

Reply:
Depends. Do we want to run as that user by default, or do we want to run as root by default?

Reply:
Root by default would match systemd behavior